docs/user-docs.md

```diff
@@ -249,6 +248,33 @@ on the **Administration** page.
     [Administration](/help/edit-administrator-settings) page.
     ```
 
+### **Administrator only feature** `{!admin-only.md!}` macro
+
+* **About:** Creates a note the feature discussed in the documentation is
+only available to organization administrators. It should be placed
+immediately after the title.
+
+* **Contents:**
+    ```.md
+    !!! warn ""
+        **Note:** This feature is only available to organization administrators.
```

Comment:
We might want slightly different language for this; perhaps "This feature can only be controlled by organization administrators"?